backend/app/routes/videos.py

```
@@ -0,0 +1,27 @@
+):
+    """비디오 요약 및 스크립트 추출"""
+    try:
+        with tempfile.NamedTemporaryFile(delete=False, suffix=os.path.splitext(file.filename)[1]) as tmp:
+            content = await file.read()
+            tmp.write(content)
+            tmp_path = tmp.name
+        transcript, summary = await video_service.process_video(tmp_path)
+        os.remove(tmp_path)
+        return VideoSummaryResponse(transcript=transcript, summary=summary)
+    except Exception as e:
+        raise HTTPException(status_code=500, detail=f"비디오 처리 실패: {e}")
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)